styles/ServicesItem.module.css

```diff
@@ -0,0 +1,62 @@
+.service {
+  display: flex;
+  flex-direction: row;
+  border-bottom: 1px solid var(--border-default);
+}
+
+.service_item__left {
+  display: flex;
+  flex-direction: column;
+  width: 33%;
+  margin:63px 0;
+}
+
+.service_left_logo {
+  width: 100px;
+  margin-bottom: 48px;
+}
+
+.services_left__link {
+  padding-top: 24px;
+  color: var(--text-default);
+  text-decoration: none;
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+  list-style-type: none;
+}
+
+.services_left__link a {
+  text-decoration: none;
+  color: var(--text-default);
+}
+
+.services_left__butoton {
+  margin-top: 48px;
+}
+
+.service_item_right {
+  display: flex;
+  width: 66%;
+  flex-direction: column;
+  border-left: 1px solid var(--border-default);
+  margin:63px 0;
+  padding-left: 48px;
+}
+
+.service_right__heading {
+  font-weight: 530;
+}
+
+.service_right__info {
+  margin-top: 48px;
+  display: grid;
+  grid-template-columns: auto auto;
+  gap: 48px;
+}
+
+.service_right__image {
+  margin-top: 48px;
+  height: 600px;
```

Comment:
<img width="1052" alt="image" src="https://user-images.githubusercontent.com/70834897/226068362-e0d5369b-b648-4326-9e81-d184d4716bc3.png">
I think it is better to delete the fixed height for avoiding the squash of the image. Or you can research on how can use fixed height and the image is not squashed, one possible way may be this one: https://www.jianshu.com/p/8a9fe8f2b7fb

Reply:
If I'm not wrong, this card is not dealing with responsive features. However, it is good to have responsive design in mind.

Reply:
Thanks I will have a try

Reply:
Thanks I will have a try